src/components/RadialProgressBar.vue

```diff
@@ -1,16 +1,45 @@
 <template>
-  <div>Hello world</div>
-  <div>making</div>
+  <div className="first">
+    <b-progress :value="value" :max="max" show-progress animated></b-progress>
+    <b-progress class="mt-2" :max="max" show-value>
+      <b-progress-bar
+        :value="value * (6 / 10)"
+        variant="success"
+      ></b-progress-bar>
+      <b-progress-bar
+        :value="value * (2.5 / 10)"
+        variant="warning"
+      ></b-progress-bar>
+      <b-progress-bar
+        :value="value * (1.5 / 10)"
+        variant="danger"
+      ></b-progress-bar>
+    </b-progress>
+
+    <b-button class="mt-3" @click="randomValue">Click me</b-button>
+  </div>
 </template>
 
 <script>
```

Comment:
I would suggest to entirely use the `<script setup> style of most modern Vue3: <https://vuejs.org/api/sfc-script-setup.html>

Reply:
We can also talk about how that would look in the code at our next meeting.